lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -823,6 +824,23 @@ def _build_block_base_path(block):
             path.append(block.display_name)
         return list(reversed(path))
 
+    @staticmethod
+    def resolve_problem_descendants(course_key, usage_key):
```

Comment:
Since problems don't have decendants:
```suggestion
    def resolve_block_descendants(course_key, usage_key):
```
Also, I found this function `yield_dynamic_block_descendants` is it helpful in this case?

Reply:
Im testing it

Reply:
@mariajgrimaldi The `yield_dynamic_block_descendants` method doesn't work for our use case because it's designed to expand dynamic blocks based on a specific `user_id`, but when we pass `user_id=None` (as required for instructor reports that need all possible problems), it fails. 
I tried it in different possible ways but unfortunately it didn't worked.

Reply:
I will apply your function name suggestion!